other-identifiers/src/main/java/org/reactome/release/otherIdentifiers/Main.java

```diff
@@ -0,0 +1,41 @@
+package org.reactome.release.otherIdentifiers;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class Main {
+
+    private static final Logger logger = LogManager.getLogger();
+    private static final String RESOURCES_DIR = Paths.get("src", "main", "resources").toString();
+
+    public static void main(String[] args) throws Exception {
+
+        String pathToConfig = args.length > 0 ? args[0] : Paths.get(RESOURCES_DIR ,"config.properties").toString();
+
+        Properties props = new Properties();
+        props.load(new FileInputStream(pathToConfig));
+
+        //Set up DB adaptor
+        String username = props.getProperty("release.database.user");
```

Comment:
If these constants ("release.database.user", "release_current.name", etc...) get used across multiple steps (and I think they do), they should probably be moved out to release-common-lib as some sort of COMMON_PROPERTY_KEYS. Not today, perhaps, but one day.

Reply:
Yea thats a nice idea, it will make the configuration files much more manageable as well